WinUIGallery/SamplePages/SampleSystemBackdropsWindow.xaml.cs

```diff
@@ -152,6 +176,16 @@ bool TrySetMicaBackdrop()
             return false; // Mica is not supported on this system
         }
 
+        bool TrySetMicaAltBackdrop()
+        {
+            return TrySetMicaBackdrop(true)
+        }
+        
+        bool TrySetMicaBackdrop()
+        {
+            return TrySetMicaBackdrop(false)
+        }
```

Comment:
I'm inclined to just inline those method calls instead of having those two functions (which don't even compile right now).

Reply:
Tnx, Done